.dockerignore

```diff
@@ -1,25 +1,3 @@
-# Ignore everything
```

Comment:
The motivation for this kind of `.dockerignore` file is to accidentally avoid adding tons of non-code to the Docker context. For example, with this change, the entire `.git` directory will be sent to Docker as part of the build context. In some cases (like Javascript projects) this can actually slow down the build process because the non-code stuff is so large.

Could we keep the old .dockerignore and instead just add a line for the end to end tests?

```
!tests/endtoend
```

Reply:
Makes sense, am totally fine with keeping the original structure and adding the new ignore exclusion. I actually had it like this originally during development but got kind of fixated on making sure the new test files didn't end up in the main image haha. I think in retrospect having a few extra test files is marginal and I think we have the unit tests files included in the main image anyways.